pkg/cmd/cli/cli.go

```diff
@@ -69,6 +69,8 @@ func NewCommandCLI(name string) *cobra.Command {
 	cmds.AddCommand(cmd.NewCmdStartBuild(f, out))
 	cmds.AddCommand(cmd.NewCmdCancelBuild(f, out))
 
+	cmds.AddCommand(cmd.NewCmdRollback(name, "rollback", f, out))
```

Comment:
Most people are going to be doing rollbacks on deployments, and deployments are the signature feature of openshift.  So rollback should be as simple as possible because it's very core.

----- Original Message -----
> > @@ -69,6 +69,8 @@ func NewCommandCLI(name string) *cobra.Command {
> >  	cmds.AddCommand(cmd.NewCmdStartBuild(f, out))
> >  	cmds.AddCommand(cmd.NewCmdCancelBuild(f, out))
> >  
> > +	cmds.AddCommand(cmd.NewCmdRollback(name, "rollback", f, out))
> 
> Only rollbacks, was arguing for clarity and standardization.
> On a second though though `rollback` would be compliant with other
> action-specific commands like `create`, `update`, `process`, etc. *And* the
> short description is good. So I'm ok with `rollback`, the only additional
> suggestion is to make the required argument part of the `Use` description
> (so it appears in the main help). Example: `Use: "rollback
> --from=<deployment>"`.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/786/files#r23774771